README.md

```diff
@@ -152,156 +47,43 @@ deployment.apps/oadp-nac-controller-manager   1/1     1            1           3
-
-```sh
-kubectl apply -f https://raw.githubusercontent.com/<org>/oadp-nac/<tag or branch>/dist/install.yaml
-```
+> **NOTE:** make sure there are no running instances of CRDs. Finalizers in those objects can fail the `undeploy` command.
 
 ## Contributing
-// TODO(user): Add detailed information on how you would like others to contribute to this project
 
-**NOTE:** Run `make help` for more information on all potential `make` targets
-
-More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
+Please check our [contributing documentation](docs/CONTRIBUTING.md) to propose changes to the repository.
 
 ## Architecture
 
-The project was generated using kubebuilder version `v3.14.0`, running the following commands
-```sh
-kubebuilder init \
-    --plugins go.kubebuilder.io/v4 \
-    --project-version 3 \
-    --project-name=oadp-nac \
-    --repo=github.com/migtools/oadp-non-admin \
-    --domain=oadp.openshift.io
-kubebuilder create api \
-    --plugins go.kubebuilder.io/v4 \
-    --group nac \
-    --version v1alpha1 \
-    --kind NonAdminBackup \
-    --resource --controller
-make manifests
-```
-> **NOTE:** The information about plugin and project version, as well as project name, repo and domain, is stored in [PROJECT](PROJECT) file
-
-To upgrade kubebuilder version, create kubebuilder structure using the current kubebuilder version and the upgrade version, using the same commands presented earlier, in two different folders. Then generate a `diff` file from the two folders and apply changes to project code.
-
-## Quality
-
-The quality checks of the project are reproduced by the continuos integration (CI) pipeline of the project. CI configuration in [`.github/workflows/ci.yml`](.github/workflows/ci.yml) file.
-
-To run all checks locally, run `make ci`.
-
-### Tests
-
-To run unit and integration tests, run
-```sh
-make simulation-test
-```
-
-TODO report, coverage and tests information
-
-### Linters and code formatters
-
-To run Go linters and check Go code format, run
-```sh
-make lint
-```
-
-To fix Go linters issues and format Go code, run
-```sh
-make lint-fix
-```
-
-Go linters and Go code formatters configuration in [`.golangci.yml`](.golangci.yml) file.
-
-To check all files format, run
-```sh
-make ec
-```
-
-Files format configuration in [`.editorconfig`](.editorconfig) file.
+For a better understanding of the project, check our [architecture documentation](docs/architecture.md).
 
-### Container file linter
-
-To run container file linter, run
-```sh
-make hadolint
-```
+## OADP version compatibility
 
-### Code generation
+OADP NAC needs OADP operator to work. The relationship between compatible versions is presented below.
 
-To check if project code was generated, run
-```sh
-make check-generate
-make check-manifests
-```
+| NAC version | OADP version |
+|-------------|--------------|
+| master      | master       |
+| v0.1.0      | v1.4.0       |
```

Comment:
maybe just add this after first release?

Reply:
We could sync those two once we have it ? 

E.g. 

OADP: v.1.4.y
NAC: v.1.4.x

so the x and y may be different but the main parts will need to be in sync?

Reply:
all other plugins (this is not exactly a plugin...) have different versions, do not know if syncing versions would be good

Reply:
to avoid confusion, matrix was removed